### ISSUE TITLE
apostrophe bug

### ISSUE THREAD

**Issue by belov38** [python-visualization/folium#961]
#### Please add a code sample or a nbviewer link, copy-pastable if possible

```python

import folium
m = folium.Map()
tt="It is marker" # Work
tt="It's marker" # Doesn't work with apostrophe (') symbol
folium.Marker(location=[0,0],tooltip=tt).add_to(m)
m
```
#### Problem description

It seems like a bug. Because map can't be rendered with a marker with a tooltip parameter which contains an apostrophe.

#### Output of ``folium.__version__``
0.6.0

**Comment by jtbaker** [comment]
Fixed in #955 (awaiting review).

I imagine there are several templates that could need to be reworked to account for this case.

@ocefpaf @Conengmo? are there any other core maintainers that could review/merge?

**Comment by ocefpaf** [comment]
> @ocefpaf @Conengmo? are there any other core maintainers that could review/merge?

Merged! Working on a new release soon. Thanks @jtbaker and @belov38!
